src/resources/Refunds.ts

```diff
@@ -63,6 +63,7 @@ export interface RefundItem<T extends Metadata = Metadata> {
     metadata?: T;
     mode: ProcessingMode;
     createdOn: string;
+    updatedOn: string;
```

Comment:
You may want to update the private SDK instead (`RefundAdminItem`) as this is an admin-only field.

Reply:
oh alright in that case I guess no changes needed here or the private sdk since it seems like `RefundAdminItem` can have the `updatedOn` field already:

```
export interface RefundAdminItem extends RefundItem, WithUpdatedOn {
    merchantId: string;
}
```